crates/jstz_cli/src/logs/trace.rs

```diff
@@ -21,16 +21,13 @@ pub async fn exec(
 
     let mut event_source = EventSource::get(&url);
 
-    let log_level = LogLevel::from_str(&log_level_str.unwrap_or_default())
-        .unwrap_or(DEFAULT_LOG_LOG_LEVEL);
-
     while let Some(event) = event_source.next().await {
         match event {
             Ok(Event::Open) => println!("Connection open with {}", url),
             Ok(Event::Message(message)) => {
                 if let Ok(log_record) = serde_json::from_str::<LogRecord>(&message.data) {
                     let LogRecord { level, text, .. } = log_record;
-                    if level <= log_level {
+                    if level <= log_level.clone().unwrap_or(DEFAULT_LOG_LOG_LEVEL) {
```

Comment:
nit: 
let's extract this into a variable outside of the while loop, that way we don't have to allocate memory for log_level in each loop :)